ccm_web/server/src/main.ts

```diff
@@ -15,11 +19,30 @@ async function bootstrap (): Promise<void> {
   const app = await NestFactory.create<NestExpressApplication>(AppModule)
 
   const configService = app.get(ConfigService)
+  const sequelize = app.get(Sequelize)
   const serverConfig = configService.get('server') as ServerConfig
   baseLogger.level = serverConfig.logLevel
 
   const isDev = process.env.NODE_ENV !== 'production'
 
+  const SequelizeStore = ConnectSessionSequelize(session.Store)
+  const sessionStore = new SequelizeStore({
+    db: sequelize,
+    table: 'Session'
```

Comment:
isn't his tablename should be `session`? all lowercase

Reply:
My current understanding is that this needs to be a string representing the name of the model. I could have let `connect-session-sequelize` generate the table for us, but it seemed more consistent to have this change represented as a migration.

If you look at the library's source code, they are looking at the `sequelize.models` object to find the model object:
https://github.com/mweibel/connect-session-sequelize/blob/e6f04dd5c5547c4a19d1de0ecb6835cae6197003/lib/connect-session-sequelize.js#L58-L62

If you log `sequelize.models` somewhere, you'll see that it's `Session: Session`.